contrib/sirupsen/logrus/example_test.go

```diff
@@ -0,0 +1,36 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package logrus
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/sirupsen/logrus"
+)
+
+func ExampleHook() {
+	// Setup logrus, do this once at the beginning of your program
+	logrus.SetFormatter(&logrus.JSONFormatter{})
+	logrus.AddHook(&DDContextLogHook{})
+
+	// Setting output to Stdout so output can be validated
```

Comment:
I think we can maybe remove this, and resort to comments only in the parts which are strictly related to using the logger with the tracer. I mean, it's obvious why this is set. It's an example that needs to be run (within the browser potentially, in godoc), so it's fine. IMHO.